src/mscorlib/shared/System/IO/Path.Windows.cs

```diff
@@ -224,41 +224,68 @@ public static ReadOnlySpan<char> GetPathRoot(ReadOnlySpan<char> path)
         /// <summary>Gets whether the system is case-sensitive.</summary>
         internal static bool IsCaseSensitive { get { return false; } }
 
+
+        /// <summary>
+        /// Returns the volume name for dos, UNC and device paths.
+        /// </summary>
         internal static ReadOnlySpan<char> GetVolumeName(ReadOnlySpan<char> path)
         {
             if (!IsPathFullyQualified(path))
```

Comment:
Can you explain why we are doing this check ?

Reply:
Volume name has no meaning if you don't know where your path is. That said, there is a corner case that I forgot about, and that is drive relative. You know the volume definitively, even if it isn't qualified.

If the path starts with just `\` or doesn't have a drive letter with `:` results from this would have no meaning. `GetPathRoot()` might be enough??

Reply:
okay understood